homeassistant/components/zoneminder/camera.py

```diff
@@ -16,25 +17,27 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def setup_platform(
+async def async_setup_platform(
```

Comment:
Why do we change to async context when the library doesn't support asyncio?

Reply:
No, that's not true.

Reply:
Let's revert these changes.

Reply:
Thank you for the insights, those changes have been reverted. Hopefully we can squash this commit noise into one are merger time...

Reply:
Yeah, we always squash when merging. :+1: